### PR TITLE
New-analyzer rollups are not invalid

### PR DESCRIPTION
**Pull request by rjmac**

### CODE REVIEW

File: soda-fountain-lib/src/main/scala/com/socrata/soda/server/highlevel/DatasetDAOImpl.scala [hunk 1]
```diff
@@ -500,41 +500,58 @@ class DatasetDAOImpl(dc: DataCoordinatorClient,
...
-                  RollupSpec(name = rollup.name, soql = "__Invalid SoQL__")
-                case ex: Exception =>
-                  log.warn(s"invalid rollup SoQL ${rollup.name} ${rollup.soql} ${ex.getMessage}")
-                  RollupSpec(name = rollup.name, soql = "__Invalid Rollup__")
+              if(rollup.isNewAnalyzer) {
+                // New-rollups don't get reworked, so we can just hand back the soql unmodified
+                RollupSpec(
+                  name = rollup.name,
+                  soql = rollup.soql
+                )
```
Comment (rjmac) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/highlevel/DatasetDAOImpl.scala:508]: Oh hrm actually this isn't quite true.  I'll have to think about this a bit more (it _is_ true that the soql per se doesn't get rewritten, but all the pointers do)

File: soda-fountain-lib/src/main/scala/com/socrata/soda/server/persistence/pg/PostgresStoreImpl.scala [hunk 2]
```diff
@@ -27,6 +27,153 @@ import org.joda.time.DateTime
...
+       |       cs.parameters
+       | FROM columns c
+       | JOIN dataset_copies dc on dc.id = c.copy_id
+       | LEFT JOIN computation_strategies cs
+       | ON c.dataset_system_id = cs.dataset_system_id AND c.column_id = cs.column_id AND c.copy_id = cs.copy_id
+       | WHERE $columnFilter
+        |dc.dataset_system_id = ? AND
+        |       dc.copy_number = ?
+        |       $deletedFilter
+        |       """.stripMargin
```
Comment (saraid) [inline soda-fountain-lib/src/main/scala/com/socrata/soda/server/persistence/pg/PostgresStoreImpl.scala:49]:
```suggestion
       | dc.dataset_system_id = ? AND
       |        dc.copy_number = ?
       |        $deletedFilter
       |        """.stripMargin
```
May as well fix it now.

**Review by saraid — APPROVED** [review]
Couldn't spot any problems, though I'm kinda irritated by all the weirdly-cased SQL that got shuffled around.

**Comment by rjmac** [comment]
> Couldn't spot any problems, though I'm kinda irritated by all the weirdly-cased SQL that got shuffled around.

Did that because of implicit-resolution (all uses of an implicit in the same file where it's defined must come after the definition)

**Pull request merged**